src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -19,7 +19,7 @@ static Https_ClientCredentialTypeTests()
         s_password = "wcfSaysHell0World!";
     }
 
-    [ConditionalFact(nameof(Root_Certificate_Installed))]
+    [ConditionalFact(nameof(Root_Certificate_Installed), nameof(Basic_Authentication_Available))]
```

Comment:
Good question.  The initial population of [ConditionalFact] across tests was based on which tests used endpoints that had an embedded EnsureRootCertificateInstalled().  Meaning that if the tests said they needed it via their endpoint's attempt to install the root cert, then we needed a matching [ConditionalFact].  It could mean the original endpoint was over-constrained.  I'll experiment with removing the root-cert condition.
/cc @iamjasonp